src/spikeinterface/curation/tests/test_curation_format.py

```diff
@@ -0,0 +1,161 @@
+    "removed_units": [31, 42],  # Can not be  in the merged_units
+}
+
+curation_ids_str = {
+    "format_version": "1",
+    "unit_ids": ["u1", "u2", "u3", "u6", "u10", "u14", "u20", "u31", "u42"],
+    "label_definitions": {
+        "quality": {"label_options": ["good", "noise", "MUA", "artifact"], "exclusive": True},
+        "putative_type": {
+            "label_options": ["excitatory", "inhibitory", "pyramidal", "mitral"],
+            "exclusive": False,
+        },
+    },
+    "manual_labels": [
+        {"unit_id": "u1", "quality": ["good"]},
+        {
+            "unit_id": "u2",
+            "quality": [
+                "noise",
+            ],
+            "putative_type": ["excitatory", "pyramidal"],
+        },
+        {"unit_id": "u3", "putative_type": ["inhibitory"]},
+    ],
+    "merged_unit_groups": [["u3", "u6"], ["u10", "u14", "u20"]],  # one cell goes into at most one list
+    "removed_units": ["u31", "u42"],  # Can not be  in the merged_units
+}
+
+# This is a failure example with duplicated merge
+duplicate_merge = curation_ids_int.copy()
+duplicate_merge["merged_unit_groups"] = [[3, 6, 10], [10, 14, 20]]
+
+
+# This is a failure example with unit 3 both in removed and merged
+merged_and_removed = curation_ids_int.copy()
+merged_and_removed["merged_unit_groups"] = [[3, 6], [10, 14, 20]]
+merged_and_removed["removed_units"] = [3, 31, 42]
+
+# this is a failure because unit 99 is not in the initial list
+unknown_merged_unit = curation_ids_int.copy()
+unknown_merged_unit["merged_unit_groups"] = [[3, 6, 99], [10, 14, 20]]
+
+# this is a failure because unit 99 is not in the initial list
+unknown_removed_unit = curation_ids_int.copy()
+unknown_removed_unit["removed_units"] = [31, 42, 99]
+
+
+def test_curation_format_validation():
+    validate_curation_dict(curation_ids_int)
+    validate_curation_dict(curation_ids_str)
+
+    with pytest.raises(ValueError):
+        # Raised because duplicated merged units
+        validate_curation_dict(duplicate_merge)
+    with pytest.raises(ValueError):
+        # Raised because Some units belong to multiple merge groups"
+        validate_curation_dict(merged_and_removed)
+    with pytest.raises(ValueError):
+        # Some merged units are not in the unit list
+        validate_curation_dict(unknown_merged_unit)
+    with pytest.raises(ValueError):
+        # Raise beecause Some removed units are not in the unit list
+        validate_curation_dict(unknown_removed_unit)
+
+
+def test_to_from_json():
+
+    json.loads(json.dumps(curation_ids_int, indent=4))
+    json.loads(json.dumps(curation_ids_str, indent=4))
+
+
+def test_convert_from_sortingview_curation_format_v0():
+
+    parent_folder = Path(__file__).parent
+    for filename in (
+        "sv-sorting-curation.json",
+        "sv-sorting-curation-int.json",
+        "sv-sorting-curation-str.json",
+        "sv-sorting-curation-false-positive.json",
```

Comment:
Where are these files???

Reply:
I think those are from the original curation format (sortingview) so they are there` src/spikeinterface/curation/tests`

Reply:
Right I forgot about them :)